client/src/services/course.ts

```diff
@@ -162,9 +162,11 @@ const courseTasksApi = new CoursesTasksApi();
 
 export class CourseService {
   private axios: AxiosInstance;
+  private apiV2: AxiosInstance;
 
   constructor(private courseId: number) {
     this.axios = globalAxios.create({ baseURL: `/api/course/${this.courseId}` });
+    this.apiV2 = globalAxios.create({ baseURL: `/api/v2/course/${this.courseId}` });
```

Comment:
We use code-generated API SDK on the client (see `src/api`). If you run `npm run openapi` in `./nestj`, it will update it and you can just import you newly created service

Reply:
Done, thanks